### PR TITLE
Add Jitter template

### PR DESCRIPTION
**Pull request by buneeIsSlo**
For #97

I've added the Jitter template. Kindly review it, @matteobruni. Thanks!

### CODE REVIEW

File: jitter/styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+  font-family: 'Space Mono', monospace;
+}
+canvas {
+  display: block;
+  vertical-align: bottom;
+}
+
+#tsparticles {
```
Comment (matteobruni) [inline jitter/styles.css:10]: Since `fullScreen` options are enabled, this CSS is not necessary.

**Review by matteobruni — CHANGES_REQUESTED** [review]
Please run `npm install` and `npm run prettier` to fix code style issues.

File: jitter/styles.css (hunk 1 above)
Comment (buneeIsSlo) [reply jitter/styles.css:10]: Understood. I’ll make the changes in a bit.

**Review by matteobruni — APPROVED** [review]
Looks good to me

**Comment by buneeIsSlo** [comment]
Thanks for the opportunity!

**Pull request merged**